### ISSUE TITLE
Add support to web SDK for endpoints that mobile supports

### ISSUE THREAD

**Issue by banool** [brim-borium/spotify_sdk#166]
This is part of a series of issues tracking progress toward bringing the Web version of this SDK in line with the mobile versions. The below (incomplete) list tracks items that haven't yet been implemented. We can update the list as we add support:

- [ ] `toggleShuffle`
- [ ] `toggleRepeat`
- [ ] `isSpotifyAppActive`
- [ ] `seekTo`
- [ ] `seekToRelativePosition`

I have investigated and confirmed that support for all of the above does exist in the Spotify API, it just needs to be hooked up. The below are functions that I'm not sure the web API supports yet (further investigation required):

- [ ] `getImage`
- [ ] `addToLibrary`

**Comment by fotiDim** [comment]
@banool They should also be already visible in the readme. Isn't one source of truth enough? If there is anything inaccurate or missing please consider updating the readme.

**Comment by banool** [comment]
This PR isn't to track what is available today, it is to track progress towards actually implementing them. The README is also vague about whether the support doesn't exist because the web API doesn't provide it vs us just not having implemented it yet.

**Comment by fotiDim** [comment]
@banool fair enough. We could adopt the [Spotify package](https://pub.dev/packages/spotify) for the missing features to minimize the effort and also see how nicely the two packages play together.

**Comment by brim-borium** [comment]
Completed in `v4.0.0-dev` (see #257). `spotify_sdk_web` now supports feature parity with mobile including `seekTo`, `seekToRelativePosition`, `setShuffle`, `setRepeatMode`, `skipToIndex`, `addToLibrary`, `removeFromLibrary`, `getLibraryState`, `getImage`, and `switchToLocalDevice`.
